packages/slice-machine/lib/builders/common/EditModal/index.js

```diff
@@ -165,8 +164,8 @@ const EditModal = ({ close, data, fields, onSave, getFieldMockConfig }) => {
             return null;
           })();
 
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-          const updatedValue = merge(initialModelValues, value);
```

Comment:
so annoying! The lodash merge method is not a pure function and changes the actual arguments on top of returning the new merged object. This means we were updating the initialModel every time, and as a consequence there no longer was a diff between values and initialValues (which would have shown the text "Save to file system")
<img width="1058" alt="image" src="https://user-images.githubusercontent.com/69144671/181260309-9f80646b-fa91-46be-9bc9-13b1520b9629.png">

Reply:
Wow super cool spot. Perhaps best to avoid lodash's merge in the future due to the side effects?